.travis.yml

```diff
@@ -27,5 +27,5 @@ before_script:
   - android-wait-for-emulator
   - adb shell input keyevent 82 &
 script:
-  - export ANDROID_NDK_HOME=/usr/local/android-sdk/ndk-bundle
-  - bash ./gradlew build connectedCheck 
+  - export ANDROID_NDK_HOME=
```

Comment:
Oh, I just looked at the travis logs. The NDK stuff is unrelated to the failure. There was something in the script that's suppose to accept the license agreement...